lesson_9/mosut_sorin/hw_lesson9_classes_mosut_sorin.py

```diff
@@ -0,0 +1,149 @@
+
+
+'''
+Build two classes of your choice that can model a real-life example.
+The class needs to meet the following requirements:
+
+    at least 5 attributes each
+
+    at least 2 methods each
+
+    one class to inherit from another
+
+As a demonstration create at least 5 instances of
+one class (preferably the child class) and
+call all the methods it holds
+
+Ex: You can have one class (Country) that has general attributes
+about countries such as area, neighbours, cities etc and
+methods related to those attributes. The second class can be a
+specific country (Romania) that has more specific attributes
+such as attractions, universities etc.
+'''
+
+# main class (parent class)
+class Automobil(object):
+    clasa = 'automobil'
+
+    def __init__(self, marca, tip, varianta, an, cilindree, putere):
+        self.marca = marca
+        self.tip = tip
+        self.var = varianta
+        self.an = an
+        self.cil = int(cilindree)
```

Comment:
When you create the class instance you can initialize the object directly with an int. By doing this the conversion is not necessary

Reply:
Also, it is encouraged to use english for variable names

Reply:
"When you create the class instance you can initialize the object directly with an int" can you be more explicit ?